sources/packages/web/src/views/student/StudentFileUploader.vue

```diff
@@ -39,30 +49,35 @@ export default {
     const reloadDocuments = ref(false);
     const formioUtils = useFormioUtils();
     const snackBar = useSnackBar();
+    const processing = ref(false);
+
     const submitForm = async (
-      submittedForm: StudentFileUploaderInfoAPIInDTO,
-      form: any,
+      form: FormIOForm<StudentFileUploaderInfoAPIInDTO>,
     ) => {
       try {
+        processing.value = true;
         const associatedFiles = formioUtils.getAssociatedFiles(form);
         const payload: StudentFileUploaderAPIInDTO = {
-          submittedForm,
+          submittedForm: form.data,
           associatedFiles,
         };
         await StudentService.shared.saveStudentFiles(payload);
-        // form reset and document list reload
+        // Form reset and document list reload.
         form.submission = {};
         reloadDocuments.value = !reloadDocuments.value;
         snackBar.success("Your documents have been submitted!");
       } catch (error) {
         let errorMessage = "An error happened while submitting your documents.";
-        if (error.response.data?.errorType === APPLICATION_NOT_FOUND) {
+        if (error.response?.data?.errorType === APPLICATION_NOT_FOUND) {
```

Comment:
This can be converted to the `catch (error: unknown)` pattern, right? Unless it is considered not part of this PR 😉